src/mon/MonClient.cc

```diff
@@ -111,14 +107,17 @@ int MonClient::get_monmap_privately()
 
   ldout(cct, 10) << "have " << monmap.epoch << " fsid " << monmap.fsid << dendl;
 
+  std::random_device rd;
```

Comment:
What's the downside to using a known good method? I realize that this is fine for our typical linux user, but it seems easy to avoid potential bugs in ports to other platforms (e.g. arm, mips, running on an ethernet drive), or using a different stdlib, etc. I see no reason to rely on potentially deterministic behavior when there's a simple alternative.